### DISCUSSION TITLE
STM32F103C8 remap USART1

### DISCUSSION THREAD

**Discussion by jjldk** [stm32duino/Arduino_Core_STM32#1318] [Q&A] [answered]
Hi all.
Is here a kind person that can show or link me to an example, on how to remap USART1 TX/RX from PA9/PA10 to PB6/PB7 ?

platform = ststm32
board = genericSTM32F103C8
framework = arduino

Thanks in advance.
/Jens

**Comment by JiriBilek** [comment]
I believe you want to create the object: `HardwareSerial Serial1(PB_7, PB_6);`
Maybe you'd need to rename it to another name to avoid predefined name Serial1.

https://github.com/stm32duino/wiki/wiki/API#hardwareserial

**Comment by fpistm** [comment, marked as answer]
Answer is here:
https://github.com/stm32duino/wiki/wiki/API#change-default-serial-instance-pins

**Comment by jjldk** [comment]
YES !!!! Thank you very much Jiri Bilek and fpistm. :-)
It now works. I changed the name of the object from Serial1 to Nextion (for my display).
The Wiki's are good reading :-)
Thanks for your great and fast help.
/Jens
